app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/DetailActivity.java

```diff
@@ -25,11 +42,17 @@
 
     public static final int ANIMATION_DURATION = 500;
 
-    Person p;
+    private Person p;
     boolean isImageZoomed;
-    Pair<Float, Float> screenDimens;
-    Pair<Float, Float> initialPicDimens;
-    Pair<Float, Float> zoomedPicTranslate;
+    private Pair<Float, Float> screenDimens;
```

Comment:
These `screenDimens` and `initialPicDimens` variables are not used after initialization.